spatialpy/Result.py

```diff
@@ -4,11 +4,18 @@
 import pickle
 import shutil
 import subprocess
+import sys
 
 import numpy
 
 from spatialpy.Model import *
-from spatialpy.VTKReader import VTKReader
+
+try:
+    import vtk
+except:
+    print('''Using fallback VTK reader. This is significantly slower than the official VTK package.
+For best performance please install the VTK Python package: \'python3 -m pip install vtk\'''')
```

Comment:
The `python3 -m pip install vtk` install command will not work on all systems.  Should we include it in the warning or have an install vtk section in the README.md?

Reply:
How about a warning recommending installation and include a link README.md for more information? If we do that there are two or three more references to recommended packages in SpatialPy that should be changed.

Reply:
Yeah, I think that would be cleaner

Reply:
please catch 'ImportError'

Reply:
Let's move this to "logging"

Reply:
Printed message is much shorter and follows other SpatialPy messages. We no longer tell the user how to install the package. As discussed earlier, will move all import error prints to logging in another PR.